src/Server.cpp

```diff
@@ -122,13 +123,29 @@ Server& Server::operator=(const Server& ref)
 void Server::run()
 {
 	RequestParser parser;
+	const int maxEvents = 10;
+	const int timeout = 5000; // epoll timeout in milliseconds
+	const int acceptedIdleTime = 10; // time in seconds until an idle connection is closed
 
 	while (true) {
-		struct epoll_event events[MAX_EVENTS];
+		struct epoll_event events[maxEvents];
```

Comment:
I know this is only changed because no Macro but funny thing:
In #22 I implemented this as std::vector to avoid C-style array. In newer standard one would just use std::array, but this way there is no array subscript access :D

Reply:
Even better!